### PR TITLE
Create recording-custom-user-agent

### PR DESCRIPTION
**Pull request by MargotLepizzera**
### What does this PR do?

This PR adds a guide to document how users can record steps with a custom user-agent.

### Preview

https://docs-staging.datadoghq.com/margot.lepizzera/new_guide/synthetics/guide/
https://docs-staging.datadoghq.com/margot.lepizzera/new_guide/synthetics/guide/recording-custom-user-agent/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/guide/recording-custom-user-agent.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+In some cases, you might need to set a specific `User-Agent` string to be able to record your browser tests steps. To do so:
+
+1. Open your application in a pop up by clickking on **Open in Popup** when on your browser test recorder.
+2. Open your Chrome Developer Tools.
+3. Click on the menu button with the three vertical dots.
+4. Select the **More tools - Network conditions** option.
+5. Go to the **Network conditions** tab and disable the **Select automatically** option.
+6. Choose **Custom** and enter the `User-Agent` string of interest.
+
+**Note:** The User-Agent string can be overriden at test execution time by adding it as a header in your test configuration.
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/recording-custom-user-agent.md:25]:
```suggestion
**Note:** The `User-Agent` string can be overridden at test execution time by adding it as a header in your test configuration. 
```

File: content/en/synthetics/guide/recording-custom-user-agent.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+further_reading:
+- link: "/synthetics/browser_tests/actions"
+  tag: "Documentation"
+  text: "Learn about steps for browser tests"
+- link: "/synthetics/browser_tests/advanced_options/"
+  tag: "Documentation"
+  text: "Configure advanced options for steps"
+---
+
+## Overview
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/recording-custom-user-agent.md:14]:
```suggestion
```

File: content/en/synthetics/guide/recording-custom-user-agent.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/recording-custom-user-agent.md:14]:
```suggestion
```

File: content/en/synthetics/guide/recording-custom-user-agent.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+In some cases, you might need to set a specific `User-Agent` string to be able to record your browser tests steps. To do so:
+
+1. Open your application in a pop up by clickking on **Open in Popup** when on your browser test recorder.
+2. Open your Chrome Developer Tools.
+3. Click on the menu button with the three vertical dots.
+4. Select the **More tools - Network conditions** option.
+5. Go to the **Network conditions** tab and disable the **Select automatically** option.
+6. Choose **Custom** and enter the `User-Agent` string of interest.
+
+**Note:** The `User-Agent` string can be overridden at test execution time by adding it as a header in your test configuration.
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/recording-custom-user-agent.md:23]:
```suggestion
**Note:** The  [default `User-Agent` string][1] can be overridden at test execution time by adding it as a header in your test configuration. 
```

File: content/en/synthetics/guide/recording-custom-user-agent.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  tag: "Documentation"
+  text: "Learn about steps for browser tests"
+- link: "/synthetics/browser_tests/advanced_options/"
+  tag: "Documentation"
+  text: "Configure advanced options for steps"
+---
+
+In some cases, you might need to set a specific `User-Agent` string to be able to record your browser tests steps. To do so:
+
+1. Open your application in a pop up by clickking on **Open in Popup** when on your browser test recorder.
```
Comment (cswatt) [inline content/en/synthetics/guide/recording-custom-user-agent.md:16]:
```suggestion
1. Open your application in a pop up by clicking on **Open in Popup** in your browser test recorder.
```

File: content/en/synthetics/guide/recording-custom-user-agent.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+2. Open your Chrome Developer Tools.
+3. Click on the menu button with the three vertical dots.
+4. Select the **More tools - Network conditions** option.
+5. Go to the **Network conditions** tab and disable the **Select automatically** option.
+6. Choose **Custom** and enter the `User-Agent` string of interest.
+
+**Note:** The  [default `User-Agent` string][1] can be overridden at test execution time by adding it as a header in your test configuration.
```
Comment (cswatt) [inline content/en/synthetics/guide/recording-custom-user-agent.md:23]:
```suggestion
**Note:** You can override the [default `User-Agent` string][1] at test execution time by adding it as a header in your test configuration. 
```

**Review by cswatt — CHANGES_REQUESTED** [review]
docs feedback

File: content/en/synthetics/guide/recording-custom-user-agent.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+further_reading:
+- link: "/synthetics/browser_tests/actions"
+  tag: "Documentation"
+  text: "Learn about steps for browser tests"
+- link: "/synthetics/browser_tests/advanced_options/"
+  tag: "Documentation"
+  text: "Configure advanced options for steps"
+---
+
+In some cases, you might need to set a specific `User-Agent` string to be able to record your browser tests steps. To do so:
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/recording-custom-user-agent.md:14]:
```suggestion
In some implementations, applications render a certain way only when using a specific `User-Agent` string (e.g., when using a mobile `User-Agent`. In this type of cases, you would need to set the `User-Agent` header to a custom string to be able to record your browser tests steps on your application. To do so:
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**